### PR TITLE
simplify Julia config

### PR DESCRIPTION
**Pull request by ChrHorn**
This simplifies the setup for the Julia language server. 
Also changes the standard indent to 4 spaces, which is recommended in the style guide. 
https://docs.julialang.org/en/v1/manual/style-guide/#Indentation

### CODE REVIEW

File: languages.toml [hunk 1]
```diff
@@ -464,21 +464,13 @@ file-types = ["jl"]
...
-        "-e",
-        """
-                using LanguageServer;
-                using Pkg;
-                import StaticLint;
-                env_path = dirname(Pkg.Types.Context().env.project_file);
-                server = LanguageServer.LanguageServerInstance(stdin, stdout, env_path, "");
-                server.runlinter = true;
-                run(server);
-        """,
```
Comment (archseer) [inline languages.toml:479]: This seemed to be the recommended config that would properly detect the project dir, as well as set up the linter: https://github.com/julia-vscode/LanguageServer.jl/wiki/Vim-and-Neovim

Comment (archseer) [reply languages.toml:479]: The docs seem to be all over the place though, with README suggesting one version, and kakoune wiki another

Comment (ChrHorn) [reply languages.toml:479]: I think most of the wiki stuff is just outdated and `runserver()` is now the recommended option.

https://github.com/julia-vscode/LanguageServer.jl/blob/master/src/runserver.jl
 `runserver()` should detect an exsting project dir automatically, linting is enabled by default.

At least for me everything seems to work fine with this setup.

Comment (ChrHorn) [reply languages.toml:479]: There seems to be an issue where I get different false positive lint messages for the new and old setup. VScode works fine. Need to dig a bit to see what's going on.

Comment (ChrHorn) [reply languages.toml:479]: Checked it again. Detection of local environments was broken with the old config, now everything should work as expected.

There are some false positives for both configs after the initial loading of a file, but they disappear after an edit.

Comment (archseer) [reply languages.toml:479]: Looks good then 👍🏻

**Comment by Seelengrab** [comment]
Naively installing the language server [as documented](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#julials) and launching helix with a julia file in a project does not start the language server for me successfully. I get the following error:

```
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "ERROR: ArgumentError: Package LanguageServer not found in current path.\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "- Run `import Pkg; Pkg.add(\"LanguageServer\")` to install the LanguageServer package.\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "Stacktrace:\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- " [1] macro expansion\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "   @ ./loading.jl:1243 [inlined]\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- " [2] macro expansion\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "   @ ./lock.jl:267 [inlined]\n"
2022-09-08T09:16:18.944 helix_lsp::transport [ERROR] err <- " [3] require(into::Module, mod::Symbol)\n"
2022-09-08T09:16:18.944 helix_lsp::transport [ERROR] err <- "   @ Base ./loading.jl:1224\n"
2022-09-08T09:16:18.956 helix_lsp::transport [ERROR] err: <- StreamClosed
2022-09-08T09:16:18.956 helix_lsp::transport [ERROR] err: <- StreamClosed
```

Which indicates to me that the language server specific environment mentioned in the documentation is not being picked up. I added `LanguageServer` to my main environment, thinking that may be solved by this, but then I get this:

```
2022-09-08T09:18:12.876 helix_term::application [ERROR] Timed out waiting for language servers to shutdown
```

What am I doing wrong?

**Comment by Seelengrab** [comment]
Naively installing the language server [as documented](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#julials) and launching helix with a julia file in a project does not start the language server for me successfully. I get the following error:

```
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "ERROR: ArgumentError: Package LanguageServer not found in current path.\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "- Run `import Pkg; Pkg.add(\"LanguageServer\")` to install the LanguageServer package.\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "Stacktrace:\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- " [1] macro expansion\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "   @ ./loading.jl:1243 [inlined]\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- " [2] macro expansion\n"
2022-09-08T09:16:18.926 helix_lsp::transport [ERROR] err <- "   @ ./lock.jl:267 [inlined]\n"
2022-09-08T09:16:18.944 helix_lsp::transport [ERROR] err <- " [3] require(into::Module, mod::Symbol)\n"
2022-09-08T09:16:18.944 helix_lsp::transport [ERROR] err <- "   @ Base ./loading.jl:1224\n"
2022-09-08T09:16:18.956 helix_lsp::transport [ERROR] err: <- StreamClosed
2022-09-08T09:16:18.956 helix_lsp::transport [ERROR] err: <- StreamClosed
```

Which indicates to me that the language server specific environment mentioned in the documentation is not being picked up. I added `LanguageServer` to my main environment, thinking that may be solved by this, but then I get this:

```
2022-09-08T09:18:12.876 helix_term::application [ERROR] Timed out waiting for language servers to shutdown
```

What am I doing wrong?

**Comment by Seelengrab** [comment]
That worked, thanks!

**Pull request merged**